test/functional/forall/atomic-ref/test-forall-atomic-ref-math-cuda.cpp

```diff
@@ -19,14 +11,17 @@
 
 #include "tests/test-forall-atomic-ref-math.hpp"
 
+#include "../test-forall-execpol.hpp"
+
 #include "../test-forall-atomic-utils.hpp"
 
 #if defined(RAJA_ENABLE_CUDA)
 using CudaAtomicForallRefMathTypes = Test< camp::cartesian_product<
-                                                                  AtomicCudaExecs,
+                                                                  CudaForallExecPols,
+                                                                  //AtomicCudaExecs,
```

Comment:
If all checks pass, please remove the previously-used policy list name that is commented out